ouroboros-consensus-cardano/src/Ouroboros/Consensus/Cardano/Block.hs

```diff
@@ -89,18 +90,25 @@ import qualified Ouroboros.Consensus.HardFork.Combinator.Util.Telescope as Teles
 
 import           Ouroboros.Consensus.Byron.Ledger.Block (ByronBlock)
 
+import qualified Shelley.Spec.Ledger.BaseTypes as Era (Shelley)
+
 import           Ouroboros.Consensus.Shelley.Ledger (ShelleyBlock)
 
+
 {-------------------------------------------------------------------------------
   The eras of the Cardano bock chain
 -------------------------------------------------------------------------------}
 
+-- | The Shelley ledger and block type itself is parameterised by an era
+-- parameter, which is in its turn parameterised by the crypto used.
+type ShelleyEra c = Era.Shelley c
+
 -- | First we had the Byron era and then the Shelley era.
 --
--- We parameterise over the crypto used in the Shelley era: @sc@.
+-- We parameterise over the crypto used in the Shelley era: @c@.
 --
 -- TODO: parameterise ByronBlock over crypto too
-type CardanoEras sc = '[ByronBlock, ShelleyBlock sc]
+type CardanoEras c = '[ByronBlock, ShelleyBlock (ShelleyEra c)]
```

Comment:
The zig zag in this definition from era to block back to era is a little confusing. Is there any chance we should rename it to `CardanoBlocks`?

Reply:
Valid point, but I think `CardanoBlocks` might be confusing w.r.t. to `CardanoBlock`? It also means more breakage. Let's revisit this when we add the next element to the list :slightly_smiling_face: